src/main/java/net/dv8tion/jda/internal/requests/RateLimiter.java

```diff
@@ -64,6 +64,11 @@ public boolean isRateLimited(Route.CompiledRoute route)
         return getRateLimit(route) != null;
     }
 
+    public int cancelRequests()
```

Comment:
This should be an abstract method shouldn't it? What's the point of making it a default-implemented method?